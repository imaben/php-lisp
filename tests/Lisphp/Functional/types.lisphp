(and (numeric? 5) (echo "5 is numeric\n"))
(or (numeric? #t) (echo "#t is not numeric\n"))
