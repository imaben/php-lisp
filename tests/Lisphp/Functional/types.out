5 is numeric
#t is not numeric
